asyncdb/drivers/sqlite.py

```diff
@@ -208,7 +208,7 @@ async def execute(self, sentence: Any, *args, **kwargs) -> Optional[Any]:
             if result:
                 await self._connection.commit()
         except Exception as err:
-            error = "Error on Execute: {err}"
+            error = f"Error on Execute: {err}"
```

Comment:
it should be fine as it is since it is done this way every where, also for the other drivers, this would be a general improvement which should be addressed in a dedicated commit if needed.